calculus.py

```diff
@@ -2,7 +2,46 @@
 calculus.py
 """
 
+import scipy.optimize as opt
+
 def dummy():
     """ dummy functions for template file
     just the same function that Dr Thomay made"""
     return 0
+
+def secant_wrapper(func, x0, x1, args=(), tol=1e-6, maxiter=50):
```

Comment:
The only linter complaint is that there are too many arguments, but i would say that in this case it makes sense to pass the args explicitly rather than compose an object to pass as the argument.